groups/restrictions.yaml

```diff
@@ -107,6 +107,7 @@ restrictions:
       - "^community@kubernetes.io$"
       - "^dev@kubernetes.io$"
       - "^github@kubernetes.io$"
+      - "^kubernetes-github-managment-alerts@kubernetes.io$"
```

Comment:
Oh one last comment.. we can probably drop the initial "kubernetes-" in the address as the domain name would now be @kubernetes.io

Reply:
removed `kubernetes-` prefix. Thanks for catching it. :)